BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -326,14 +314,14 @@ public bool BacklightCompensation
         {
             get
             {
-                CheckOptionSupported(RealSense2API.Option.BACKLIGHT_COMPENSATION, BacklightCompensationDesc.Name, RealSense2API.SensorName.COLOR);
-                return RealSense2API.GetOption(_pipeline, RealSense2API.SensorName.COLOR, RealSense2API.Option.BACKLIGHT_COMPENSATION) == 1.0f ? true : false;
+                CheckOptionSupported(Option.BacklightCompensation, BacklightCompensationDesc.Name, SensorName.COLOR);
+                return GetOption(SensorName.COLOR, Option.BacklightCompensation) == 1.0f ? true : false;
```

Comment:
BTW: you can omit the `? true : false`, because it means: if true, then true, else false

Reply:
fixed